ntoskrnl/ex/init.c

```diff
@@ -1579,17 +1580,30 @@ Phase1InitializationDiscard(IN PVOID Context)
                                WINDOWS_NT_INFO_STRING_PLURAL :
                                WINDOWS_NT_INFO_STRING,
                                &MsgEntry);
+    if (NT_SUCCESS(MsgStatus))
+    {
+        InfoStringFormat = (PCHAR)MsgEntry->Text;
+    }
+    else
+    {
+        InfoStringFormat = (KeNumberProcessors > 1) ?
```

Comment:
There's no reason RtlFindMessage should fail. Duplicating every English string in case the real string can't be loaded is not useful.

Reply:
Ah, I see you just moved that up from below. Useless code churn. Avoid.

Reply:
Well, this file checks all `RtlFindMessage()` calls for failure. Out of my scope.
Though current code is enough as a fallback, it seems odd to me not to do it 1-for-1 string. Without at least a comment.
Then: new code? a comment? as is?

Reply:
It's a fallback. It doesn't need a plural form.

Reply:
Reverted. `(s)` added instead...